README.md

```diff
@@ -60,17 +60,6 @@ Currently, following features are supported:
   tenant-specific roles (`TENANT_WRITER`, `TENANT_READER`) will be able to
   access only their own tenant's data.
 
-DAL uses 4 database roles/users to perform all operations:
-
-- `TENANT_READER` - has read access to its tenant's data
-- `READER` - has read access to all tenants' data
-- `TENANT_WRITER` - has read & write access to its tenant's data
-- `WRITER` - has read & write access to all tenants' data
-
-DAL allows to map a user's service role to the DB role that will be used for
-that user. If a user has multiple service roles which map to several DB roles,
-the DB role with the most extensive privileges will be used (see `Less()` method
-for `DbRoleSlice` for details).
```

Comment:
Let's have a _quickstart.md_ file in the future, where we would have some code demonstrating the usage of the library. We can describe the meaning of these DB roles in the comments in that file